{{cookiecutter.project_slug}}/_/development/python/Dockerfile

```diff
@@ -41,6 +41,6 @@ USER 1001:0
 RUN DJANGO_SECRET_KEY=collectstatic \
     python manage.py collectstatic --noinput --link
 {% endif %}
-ENTRYPOINT ["deployment/application/entrypoint.sh"]
+ENTRYPOINT ["entrypoint.sh"]
```

Comment:
This is a painful mistake. :worried: The call needs to have a path if that is a local script. Otherwise the shell tries to find a command named that was in the `PATH`. Must look like that:

```Dockerfile
ENTRYPOINT ["./entrypoint.sh"]
```

This went unnoticed, because our field test doesn't report errors like that. For Kubernetes this is a simple situation of "let's try again", and it tries restarting the Pod that fails to start up. (This even starts to exhaust the quota limits for resources after a while, which consequently buries the original error message and make the actual problem hard to discover.)

We need to integrate a way to discover this situation and notify developers right when the problem occurs the first time! :bomb: